services/scabbard/libscabbard/src/service/v3/consensus/consensus_runner/mod.rs

```diff
@@ -109,7 +109,7 @@ where
                 }
             };
 
-            let (event_id, event) = event.deconstruct();
+            let (event_id, event) = (event.id, event.record);
```

Comment:
We could still keep the deconstruct method.

Reply:
Added a deconstruct method for Identified<T>